Model/Currency.php

```diff
@@ -208,6 +208,6 @@ protected function getConsumerIp()
      */
     protected function checkLocalIP($ip)
     {
-        return in_array($ip, ['localhost','127.0.0.1']);
+        return in_array($ip, ['localhost','127.0.0.1','172.19.0.1','172.25.0.7','172.25.0.1']);
```

Comment:
could you just add a comment indicating what the extra IP addresses are for? I think localhost and 127.0.0.1 are self-explanatory, but it would help to have right here what the other IPs are for.